go/adbc/driver/bigquery/statement.go

```diff
@@ -248,7 +259,23 @@ func (st *statement) SetOption(key string, v string) error {
 		} else {
 			return err
 		}
-
+	case OptionStringIngestPath:
+		st.isIngest = true
+		st.ingestPath = v
+	case OptionStringIngestFileDelimiter:
+		st.ingestFileDelimiter = v
+	case adbc.OptionKeyIngestMode:
+		switch v {
+		case adbc.OptionValueIngestModeCreateAppend, adbc.OptionValueIngestModeAppend:
```

Comment:
I'm not 100% sure, but `adbc.OptionValueIngestModeCreateAppend` should be `bigquery.WriteEmpty`. If that's not the case, I would prefer we didn't support the mode here and failed instead.